pkgs/development/tools/misc/dura/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgs }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dura";
+  version = "unstable-2022-01-07";
+
+  src = fetchFromGitHub {
+    owner = "tkellogg";
+    repo = "dura";
+    rev = "a41863879772650f2538a978ca926357bae674db";
+    sha256 = "sha256-eBgD1rBSm314I9/f1q/tOzJHjMrftbzow2HTz37njak=";
+  };
+
+  cargoSha256 = "sha256-pBi+6QfFxGjNJ+13YnMUDpDIGTtRoJhEGazrpKb04Cg=";
+
+  buildInputs = with pkgs; [
+    openssl
+  ];
+
+  nativeBuildInputs = with pkgs; [
+    pkg-config
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "You shouldn't ever lose your work if you're using Git.";
+    longDescription = ''
+    Dura is a background process that watches your Git repositories and commits
```

Comment:
Indent this, two spaces, 80-column limit.

Reply:
Fixed, thanks !